content/en/contribute/_index.md

```diff
@@ -7,111 +7,136 @@ menu:
-4) [Help translate the glossary](#help-translate-the-glossary)
+4) [Localize the glossary](#help-localize-the-glossary)
+
+## CNCF glossary overview 
+
+The goal of this glossary is to simplify the cloud native space — which is notorious for its complexity — and thus make it more accessible to people. 
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+How you can participate in this project depends on your level of cloud native expertise. 
+Simplifying complex concepts requires a deep knowledge of the topic. 
+Therefore, to contribute new terms, you must be proficient in them. 
+Contributors are typically engineers who have worked with these technologies for some time or academics focused on cloud native. 
+
+That know-how is required because explaining complex concepts in simple words is _really_ hard. And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity results from hard work and collaboration between cloud native experts. 
+
+If you are not a cloud native expert yet but still want to contribute, we recommend teaming up with someone who is. 
+Once the expert is confident that the term accurately describes the concept, you are ready for your first Glossary contribution.
+
+The localization effort is where beginners proficient in another language can make valuable contributions to the Glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read this guide's [Help Localize the glossary](#help-localize-the-glossary) section to learn how to get started. 
+
+## Before you start
+
+Before beginning your Glossary contributor journey, be sure to complete the following steps:
+
+1. Create a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account), if you don't have one already. 
+2. Sign the [Contributor License Agreement](https://docs.linuxfoundation.org/lfx/easycla/v2-current/contributors) (CLA). 
+3. [Verify your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)
+4. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits. 
```

Comment:
Hi @CathPag @valeriahhdez 
I'm not sure step 3 is an actionable step at this early stage. I believe that commit signature verification can only be done after a contributor has pushed a commit (or opened a PR) to GitHub repository. I think we can skip the step 3 and utilize the link only.

```suggestion
3. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "[Verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)" status on your commits. 
```

Reply:
The link I provided in this step is to verify commit signatures with SSH or GPG keys, which can be completed using the GitHub interface before making any commit or push. Maybe you're referring to the -S flag used to verify with the git commit command?
We could add that option too and make it clear that signing commits this way is only required at the moment of committing the changes.

Reply:
Hi @valeriahhdez 
Thank you for the clarification. I misunderstood. I thought the step is to check if a commit contains a verified signature. 
I think we can merge step 3, 4 into one since they are very related subjects (related with commit signature). It is to simplify the steps.

Applying my comment is up to you.
LGTM as is :) 

```suggestion
3. [Configure your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) and enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits. 
```